index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
Make sure to remove empty lines. Get into the habit of this early on because eventually points will be taken away from your overall score.